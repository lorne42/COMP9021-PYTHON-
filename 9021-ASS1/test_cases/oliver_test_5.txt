You have

met a group of 6 islander. Their
names are Sirs Uri, Beatrix, Samuel, Yuri, Connor and Vincent.

Sir Beatrix  says: "Sir Samuel is a Knave." "Sir Uri is a Knave," Sir Beatrix speaks 
out again. Sir Connor tells you: "Sir Yuri is a Knave!" Sir 	Uri says: "Sir Samuel and I are Knights."
Sir Vincent says: "Sir Connor is a Knight!" "Sir 
Vincent is a Knave," Sir Yuri wisphered. Sir Samuel talks: "Sir Connor and I are Knights." What
do
you 
	think?