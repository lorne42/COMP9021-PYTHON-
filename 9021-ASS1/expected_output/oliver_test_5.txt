The Sirs are: Beatrix Connor Samuel Uri Vincent Yuri
There are 3 solutions.